src/fidesctl/core/export.py

```diff
@@ -163,19 +164,20 @@ def generate_system_records(
         )
     ]
 
-    for system in server_system_list:
+    for system in server_resources["system"]:
         third_country_list = ", ".join(system.third_country_transfers or [])
         data_protection_impact_assessment = (
             get_formatted_data_protection_impact_assessment(
                 system.data_protection_impact_assessment.dict()
             )
         )
         for declaration in system.privacy_declarations:
-            data_use = get_formatted_data_use(url, headers, declaration.data_use)
+            data_use = formatted_data_uses[declaration.data_use]
             data_categories = declaration.data_categories or []
-            data_subjects = get_formatted_data_subjects(
-                url, headers, declaration.data_subjects
-            )
+            data_subjects = [
+                formatted_data_subjects[data_subject_fides_key]
```

Comment:
If the `data_subject`  was missing, it would error using the `apply` command before we can pull these from the server, so this should be set as well 👍🏽